syne_tune/blackbox_repository/blackbox.py

```diff
@@ -45,7 +48,8 @@ def objective_function(
         or if it has a single fidelity in its fidelity space. In the latter case, all fidelities are returned in form
         of a tensor with shape (num_fidelities, num_objectives).
         :param seed:
-        :return: dictionary of objectives evaluated or tensor with shape (num_fidelities, num_objectives) if no fidelity
+        :return: dictionary of objectives evaluated or tensor with shape
+            (num_fidelities, num_objectives) if no fidelity
         was given.
```

Comment:
add this part of the docstring to the previous line

Reply:
OK